backend/lib/messaging/sending.ts

```diff
@@ -103,29 +109,28 @@ async function sendMultipleTousABordNotificationEmails(limit: number) {
   console.log(results)
 }
 
-async function processSingleEmail(
-  emailType: EmailCategory,
-  followupId: string
-) {
+async function processSingleEmail(emailType: EmailType, followupId: string) {
   const followup: Followup | null = await Followups.findById(followupId)
   if (!followup) {
     throw new Error("Followup not found")
   }
 
-  let emailPromise: Promise<void>
+  let emailPromise: Survey | Followup
 
   switch (emailType) {
-    case EmailCategory.SimulationResults:
-      emailPromise = followup.sendSimulationResultsEmail()
+    case EmailType.SimulationResults:
+      emailPromise = await sendSimulationResultsEmail(followup)
```

Comment:
C’est plus vraiment un email promise c’est c’est await avant d’être assigné, après on a p.e. plus besoin de d'assigner on peut juste enlever la variable `emailPromise` et faire 
```suggestion
     await sendSimulationResultsEmail(followup)
```

Reply:
Et du coup enlever les lignes 
```
  const email = await emailPromise
  console.log("Email sent", email)
```

(en plus c'était pas un email mais soit un survey soit un followup)

Reply:
C'est juste